app/views/calendar/_create_event.erb

```diff
@@ -0,0 +1,50 @@
+<div class="modal fade"
+     id="create-event-modal"
+     tabindex="-1"
+     role="dialog"
+     aria-labelledby="create-event-modal-label">
+  <div class="modal-dialog" role="document">
+    <%= form_for :event, url: events_path, remote: true, html: {id: 'create-event-form'} do |e| %>
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+          <h1>Create Event</h1>
+        </div>
+        <div class="modal-body">
+
+          <div class="alert alert-danger" id="create-event-error"></div>
+
+          <section class="form-group">
+            <%= label_tag :description, 'Description' %>
+            <%= e.text_field :description, class: 'form-control' %>
+          </section>
+
+          <section class="row">
+            <div class="col-md-5">
+              <%= render partial: 'datepicker', locals: {event: e, name: :start_time, text: 'Start Time', id: 'start-time-picker'} %>
```

Comment:
make sure this partial is displayed only once otherwise this: `id: 'start-time-picker'` wont work correctly